custom_components/tuya_local/devices/zemismart_spm02v25_energymeter.yaml

```diff
@@ -0,0 +1,196 @@
+name: Energy meter
+products:
+  - id: iwn0gpzz
+    manufacturer: Zemismart
+    model: "SPM02 Zigbee V2.5"
+entities:
+  - entity: sensor
+    name: Total Energy
+    class: energy
+    state_class: total_increasing
+    icon: mdi:home-import-outline
+    dps:
+      - id: 1
+        type: integer
+        name: sensor
+        unit: kWh
+        mapping:
+          - scale: 100
+
+  - entity: sensor
+    name: Phase A Raw Data
+    category: diagnostic
+    icon: mdi:code-json
+    dps:
+      - id: 6
+        type: string
+        name: sensor
+        optional: true
+
+  - entity: sensor
+    name: Fault Status
```

Comment:
Use the standard problem binary_sensor pattern that most device configs are already using. This gives a single binary sensor that is easy to trigger notifications or other automations off for any fault condition, and still leaves the raw fault code and description as attributes (when available as below) for advanced use.